src/emulator/gui/src/game_selector.cpp

```diff
@@ -179,22 +181,98 @@ void draw_game_selector(GuiState &gui, HostState &host) {
                     }
                     if (ImGui::IsMouseClicked(0))
                         selected[0] = true;
+                    if (gui.title_id != game.title_id)
+                        gui.title_id = game.title_id;
                 }
+
+                if (gui.title_id == game.title_id) {
+                    if (ImGui::BeginPopupContextItem("#game_context_menu")) {
+                        if (ImGui::MenuItem("Boot", game.title.c_str()))
+                            selected[0] = true;
+                        if (ImGui::BeginMenu("Copy Game Info")) {
```

Comment:
"Copy game info"

What is this weird uppercase style

Reply:
hmm yes you have right haha

Reply:
This is the usual with all the other menu and option or all the first letter of the word are capitalized